acceptance/tests/reports/event_query_with_read_db.rb

```diff
@@ -32,7 +32,7 @@ class { '::postgresql::server':
   step "Copy db schema to the new puppetdb2 database" do
 
     duplicate_db_command = <<DUPE
-    su postgres -c "PATH=/usr/pgsql-9.3/bin:/usr/lib/postgresql/9.3/bin:$PATH pg_dump puppetdb | sed -e 's/OWNER TO.*;/OWNER TO puppetdb2;/i' | psql puppetdb2"
+    su postgres -c "PATH=/usr/pgsql-9.4/bin:/usr/lib/postgresql/9.4/bin:$PATH pg_dump puppetdb | sed -e 's/OWNER TO.*;/OWNER TO puppetdb2;/i' | psql puppetdb2"
```

Comment:
Not critical, but wouldn't mind wrapping this while we're here, i.e.

    su postgres -c "PATH=/usr/pgsql-9.4/bin:/usr/lib/postgresql/9.4/bin:$PATH
      pg_dump puppetdb \
      | sed -e 's/OWNER TO.*;/OWNER TO puppetdb2;/i' \
      | psql puppetdb2" 

or whatever indentation style we prefer...